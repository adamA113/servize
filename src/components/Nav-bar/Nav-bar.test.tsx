import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Nav-bar';
import '@testing-library/jest-dom';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() });
it('should render correctly with no props', () => {
    const component = shallow(<Navbar />);

    expect(component).toMatchSnapshot();
});
// it('should render banner text correctly with given strings', () => {
//     const strings = ['one', 'two'];
//     const component = shallow(<Navbar list={strings} />);
//     expect(component).toMatchSnapshot();
// });