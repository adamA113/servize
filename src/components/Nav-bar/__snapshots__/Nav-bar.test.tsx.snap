// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with no props 1`] = `
<header
  id="nav-bar"
>
  <nav>
    <div
      className="menu-icon"
    >
      <i
        className="fa fa-bars fa-2x"
      />
    </div>
    <div
      className="logo"
    >
      <a
        href="#"
      >
        Servize
      </a>
    </div>
    <div
      className="menu"
    >
      <ul>
        <li>
          <a
            href="#"
          >
            How it Works
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Browse Jobs
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Language
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Become a Seller
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Sign In
          </a>
        </li>
        <li>
          <Link
            to="/user/forgot-password"
          >
            Join
          </Link>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
